autoload/lsp/internal/work_done_progress.vim

```diff
@@ -0,0 +1,45 @@
+let s:lsp_progress = {
+      \ 'server': '',
+      \ 'token': '',
+      \ 'title': '',
+      \ 'messages': '',
+      \ 'percentage': 100,
+      \ }
+
+function! lsp#internal#work_done_progress#_enable() abort
+    if !g:lsp_work_done_progress_enabled | return | endif
+
+    let s:Dispose = lsp#callbag#pipe(
```

Comment:
also add `s:enabled` check like https://github.com/prabirshrestha/vim-lsp/pull/978/files so someone doesn't accidentally subscribe multiple times. Primarily used for unit testing.

Reply:
Thank you for advice.
I fixed it.